UIforETW/Utility.h

```diff
@@ -76,30 +88,24 @@ int64_t GetFileSize(const std::wstring& path);
 void SetClipboardText(const std::wstring& text);
 std::wstring GetClipboardText();
 
-enum WindowsVersion
-{
-	kWindowsVersionXP,
-	kWindowsVersionVista,
-	kWindowsVersion7,
-	kWindowsVersion8,
-	kWindowsVersion8_1,
-	kWindowsVersion10,
-};
+std::wstring GetEnvironmentVariableString(_In_z_ PCWSTR variable);
+std::string GetEnvironmentVariableString(_In_z_ PCSTR variable);

```

Comment:
I'd need to change `CUIforETWDlg::SetSymbolPath` to use wide character versions. Should I?